src/lib/Tab/Dashboard/PagerContentToDataMapper.php

```diff
@@ -10,101 +10,51 @@
 
 use eZ\Publish\API\Repository\ContentService;
 use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\API\Repository\LanguageService;
 use eZ\Publish\API\Repository\UserService;
 use eZ\Publish\Core\Helper\TranslationHelper;
 use eZ\Publish\Core\MVC\Symfony\Locale\UserLanguagePreferenceProviderInterface;
-use EzSystems\EzPlatformAdminUi\Specification\ContentIsUser;
-use EzSystems\EzPlatformAdminUi\Specification\UserExists;
-use Pagerfanta\Pagerfanta;
 
-class PagerContentToDataMapper
+/**
+ * @deprecated This class has been moved to \EzSystems\EzPlatformAdminUi\Search\PagerContentToDataMapper
+ */
+class PagerContentToDataMapper extends \EzSystems\EzPlatformAdminUi\Search\PagerContentToDataMapper
 {
-    /** @var ContentService */
+    /** @var \eZ\Publish\API\Repository\ContentService */
     protected $contentService;
 
-    /** @var ContentTypeService */
+    /** @var \eZ\Publish\API\Repository\ContentTypeService */
     protected $contentTypeService;
 
-    /** @var UserService */
+    /** @var \eZ\Publish\API\Repository\UserService */
     protected $userService;
 
-    /** @var UserLanguagePreferenceProviderInterface */
-    private $userLanguagePreferenceProvider;
-
-    /** @var \eZ\Publish\Core\Helper\TranslationHelper */
-    private $translationHelper;
-
     /**
-     * @param ContentService $contentService
-     * @param ContentTypeService $contentTypeService
-     * @param UserService $userService
-     * @param UserLanguagePreferenceProviderInterface $userLanguagePreferenceProvider
+     * @param \eZ\Publish\API\Repository\ContentService $contentService
+     * @param \eZ\Publish\API\Repository\ContentTypeService $contentTypeService
+     * @param \eZ\Publish\API\Repository\UserService $userService
+     * @param \eZ\Publish\Core\MVC\Symfony\Locale\UserLanguagePreferenceProviderInterface $userLanguagePreferenceProvider
      * @param \eZ\Publish\Core\Helper\TranslationHelper $translationHelper
+     * @param \eZ\Publish\API\Repository\LanguageService $languageService
      */
     public function __construct(
         ContentService $contentService,
+        TranslationHelper $translationHelper,
         ContentTypeService $contentTypeService,
         UserService $userService,
         UserLanguagePreferenceProviderInterface $userLanguagePreferenceProvider,
-        TranslationHelper $translationHelper
+        LanguageService $languageService
```

Comment:
The safest approach for BC here is not touching this class at all. Since it's no longer used, there's no issue with that. It _is_ no longer used, right? :)

the only change to the constructor should be 
```php
@trigger_error(
    sprintf(
        '%s is deprecated, use %s instead', 
        __CLASS__, 
        \EzSystems\EzPlatformAdminUi\Search\PagerContentToDataMapper::class
    ),
    E_USER_DEPRECATED
)
```
(please import new class as an alias for readability)

Reply:
That's good idea, but new templates required some changes in this mapper to work. Maybe we should add them also here, in case someone uses the deprecated mapper?